specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/JavaComponents.json

```diff
@@ -360,6 +360,43 @@
     }
   },
   "definitions": {
+    "ScgRoute": {
+      "description": "Spring Cloud Gateway route definition",
+      "type": "object",
+      "required": [
+        "id",
+        "uri"
+      ],
+      "properties": {
+        "id": {
+          "description": "Id of the route",
+          "type": "string"
+        },
+        "uri": {
+          "description": "Uri of the route",
```

Comment:
Declare as format = uri

Reply:
It can be some non url values like "lb:customer-service", which means the gateway will ask the real ip/uri from a service discovery service. So I think string can better surve the case.

Reply:
And it maybe consist of other formated "uri"s that I'm not aware of, if the customer is using some other service discovery component to work with the gateway.

Reply:
Hmm. Is "lb:customer-service" an URI? I guess the answer is maybe, 'lb' is not a well-known scheme. But... maybe its well known in SCG context? I have no idea! :-) (yet)

But more to the point, you now appear to be claiming that it is an URI (in the description) and yet also that its not an URI. Surely only one of these can be the truth?